lib/tictactoe.rb

```diff
@@ -26,63 +19,72 @@ def print_board()
 
   def play_game
     @move_count = 0
-    @current_player = nil
-    while @move_count < 10 && !won?
-      @current_player = @move_count.even? ? player1 : player2
-      print "#{@current_player.name} Enter a row: "
-      row = gets.chomp.to_i
+    current_player = nil
+    has_won = false
+
+    while !draw? && !has_won
+      current_player = @move_count.even? ? player1 : player2
+
+      row = get_input(current_player, 'row').to_i
       next unless valid_input?(row)
```

Comment:
- [ ] One part of this method could be in the main, so you move all the prints/puts there. Remember in some methods you can just return something, and print the method in the main file.